.github/workflows/main.yml

```diff
@@ -54,20 +54,45 @@ jobs:
           asset_name: hbase-table-provisioner-${{ needs.get-publish-version.outputs.publish-version }}.jar
           asset_content_type: application
 
-  build-app:
+#  build-app:
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@master
+#      - name: build and test
+#        run: make integration-all
+#      - name: java setup
+#        uses: actions/setup-java@v1
+#        with:
+#          java-version: 1.8
+#      - name: gradle build
+#        run: make local-all
+#      - name: Upload app
+#        uses: actions/upload-artifact@v2
+#        with:
+#          name: built-app
+#          path: build/libs
+
+  publish-docker:
     runs-on: ubuntu-latest
+#    needs: [get-publish-version, build-app]
+    needs: get-publish-version
     steps:
-      - uses: actions/checkout@master
-      - name: build and test
-        run: make integration-all
-      - name: java setup
-        uses: actions/setup-java@v1
-        with:
-          java-version: 1.8
-      - name: gradle build
-        run: make local-all
-      - name: Upload app
-        uses: actions/upload-artifact@v2
+#      - uses: actions/checkout@master
+#      - name: Download built app
+#        uses: actions/download-artifact@v2
+#        with:
+#          name: built-app
+#          path: build
+      - name: show dir
+        run: ls -la && ls -la build
+      - name: Get release version
+        id: get_version
+        run: echo ::set-env name=RELEASE_VERSION::$(echo ${GITHUB_REF:10})
+      - name: Publish to DockerHub
+        uses: elgohr/Publish-Docker-Github-Action@master
         with:
-          name: built-app
-          path: build/libs
+          name: ${{ env.IMAGE_NAME }}
```

Comment:
this isn't set, see: https://github.com/dwp/dataworks-repo-template-docker/blob/master/.github/workflows/main.yml#L12

Reply:
Nice spot, thanks